EventManager.csproj

```
@@ -4,7 +4,7 @@
     <TargetFrameworks>netstandard2.0;</TargetFrameworks>
     <RootNamespace>EventManager</RootNamespace>
     <PackageId>Cecropia.Utilities.EventManager</PackageId>
-    <Version>1.2.0</Version>
+    <Version>2.0</Version>
```

Please add a patch version so that we respect semantic versioning:

```suggestion
    <Version>2.0.0</Version>
```

---

fixed too